person.rb

```diff
@@ -0,0 +1,26 @@
+# Defines the Person class
+class Person
+  def initialize(id, age, name = 'Unknown', parent_permission: true)
+    @id = id
```

Comment:
- [OPTIONAL] I think generating the `id` automatically when a new person is got created will be much better. In that case, you might not need to pass `id` as an argument in the initialize method.

See example:
```ruby
def initialize(age, name = "Unknown", parent_permission = true)
    @id = Random.rand(1..1000)
    @age = age
    @name = name
    @parent_permission = parent_permission
  end
```